src/js/player.js

```diff
@@ -99,7 +99,11 @@ vjs.Player = vjs.Component.extend({
 
     if (options['plugins']) {
       vjs.obj.each(options['plugins'], function(key, val){
-        this[key](val);
+        try {
```

Comment:
I was trying to figure out what the right way to go was here but, in my case, my video.js player is just an empty shell without the plugin and I want to abort further loading. It sounds like logging it would not help me determine if the video.js player (and required plugin) is broken?